File: CPP/Program/Mahasiswa.cpp
//LIBABRY
#include <iostream>
#include <string>

//STANDAR NAMESPACE
using namespace std;

//DEKLARASI KELAS
class Mahasiswa
{ 
    //ATRIBUT PRIVAT
    private:
        string nama;
        int nim;
        string prodi;
        string fakultas;

    //ATRIBUT PUBLIK
    public:

        //KONSTRUKTOR
        Mahasiswa()
        {

        }

        Mahasiswa(string nama, int nim, string prodi, string fakultas)
        {
            this->nama = nama;
            this->nim = nim;
            this->prodi = prodi;
            this->fakultas = fakultas;
        }

        //GETTER DAN SETTER
        string get_nama()
        {
            return this->nama;
        }

        void set_nama(string nama)
        {
            this->nama = nama;
        }

        int get_nim()
        {
            return this->nim;
        }

        void set_nim(int nim)
        {
            this->nim = nim;
        }

        string get_prodi()
        {
            return this->prodi;
        }

        void set_prodi(string prodi)
        {
            this->prodi = prodi;
        }

        string get_fakultas()
        {
            return this->fakultas;
        }

        void set_fakultas(string fakultas)
        {
            this->fakultas = fakultas;
        }

        //DESTRUKTOR
        ~Mahasiswa()
        {

        }
};

File: Python/Program/Mahasiswa.py
class Mahasiswa:
    ## private class
    __nama = ""
    __nim = ""
    __prodi = ""
    __fakultas = ""

    #KONSTRUKTOR
    def __init__(self, nama, nim, prodi, fakultas):
        self.__nama = nama
        self.__nim = nim
        self.__prodi = prodi
        self.__fakultas = fakultas

    #GETTER SETTER
    def get_nama(self):
        return self.__nama

    def set_nama(self, nama):
        self.__nama = nama

    def get_nim(self):
        return self.__nim

    def set_nim(self, nim):
        self.__nim = nim

    def get_prodi(self):
        return self.__prodi

    def set_prodi(self, prodi):
        self.__prodi = prodi

    def get_fakultas(self):
        return self.__fakultas

    def set_fakultas(self, fakultas):
        self.__fakultas = fakultas